shared/src/business/useCaseHelper/initialFilingDocuments/updateInitialFilingDocuments.js

```diff
@@ -92,9 +92,10 @@ exports.updateInitialFilingDocuments = async ({
   caseEntity,
   caseToUpdate,
 }) => {
+  const PETITION_KEY = 'petitionFile';
```

Comment:
Nice, I wonder if this string is used elsewhere in the code and if this would be worth moving to `EntityConstants`

Reply:
@mrinsin there is a use in `CaseInternal.js` for validation, but there are several similar strings in use there too. Maybe a separate PR for that change if we want it? I don't feel strongly either way.

Reply:
I don't feel too strongly about the changes being in this PR! 👍 to different PR